### PR TITLE
ci: preserve symlinks while zipping macos app

### PR DESCRIPTION
**Pull request by x1ddos**
See build docs https://github.com/digitalbitbox/bitbox-wallet-app/blob/673708a/docs/BUILD.md#signing--notarization and #903.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ jobs:
       - name: Archive app
         run: >
           pushd ~/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/qt/build/osx;
-          zip -r ${{github.workspace}}/BitBoxApp-macos.zip BitBox.app;
+          zip -y -r ${{github.workspace}}/BitBoxApp-macos.zip BitBox.app;
```
Comment (benma) [inline .github/workflows/ci.yml:91]: Actually we should use `ditto`

https://github.com/digitalbitbox/bitbox-wallet-app/pull/903#discussion_r448808691

as `zip` seems to modify the app bundle in undesired ways. `ditto` behaves like the Finder when compressing.

Comment (x1ddos) [reply .github/workflows/ci.yml]: Done.

**Comment by x1ddos** [comment]
PTAL

**Comment by x1ddos** [comment]
Also, this now reports smaller size indeed:

```
Finished uploading artifact BitBoxApp-macos-4dcc80d0b8a8ce8fedb20426ce5f5bd546cf6a24.zip. Reported size is 110714984 bytes
```

from https://github.com/digitalbitbox/bitbox-wallet-app/pull/907/checks?check_run_id=830161403

**Pull request merged**